amanuensis/resources/admin/admin_state.py

```diff
@@ -59,13 +60,59 @@ def update_project_state(project_id, state_id):
 
         request_schema = RequestSchema(many=True)
         consortium_statuses = config["CONSORTIUM_STATUS"]
-        requests = udm.update_project_state(
-            session, requests, state, consortium_statuses, project_id
-        )
+        consortiums = []
+        for request in requests:
+            consortium = request.consortium_data_contributor.code
+            state_timestamp = list(
+                zip(
+                    [states.state.code for states in request.request_has_state],
+                    [states.state.create_date for states in request.request_has_state],
```

Comment:
Is this maintaining the order between state and date? Like doing `[(states.state.code,states.state.create_date) for states in request.request_has_state]`

Reply:
Yes, this will maintain the order.